grafana-plugin/src/containers/MaintenanceForm/MaintenanceForm.tsx

```diff
@@ -49,34 +54,151 @@ export const MaintenanceForm = observer((props: MaintenanceFormProps) => {
+                      onChange={(value) => {
+                        field.onChange(value);
+                      }}
+                    />
+                  </Field>
+                )}
+              />
+              <Controller
+                name="mode"
+                control={control}
+                rules={{ required: 'Mode is required' }}
+                render={({ field }) => (
+                  <Field
+                    label="Mode"
+                    description="Choose maintenance mode: Debug (test routing and escalations without real notifications) or Maintenance (group alerts into one during infrastructure work)."
+                    invalid={!!errors.mode}
+                    error={errors.mode?.message}
+                  >
+                    <Select
+                      placeholder="Choose mode"
+                      value={field.value}
+                      menuShouldPortal
+                      options={[
+                        {
+                          value: MaintenanceMode.Debug,
+                          label: 'Debug (silence all escalations)',
+                        },
+                        {
+                          value: MaintenanceMode.Maintenance,
+                          label: 'Maintenance (collect everything in one alert group)',
+                        },
+                      ]}
+                      onChange={(option: SelectableValue) => {
+                        field.onChange(option.value);
+                      }}
+                    />
+                  </Field>
+                )}
+              />
+              <Controller
+                name="duration"
+                control={control}
+                rules={{ required: 'Duration is required' }}
+                render={({ field }) => (
+                  <Field
+                    label="Duration"
+                    description="Specify duration of the maintenance"
+                    invalid={!!errors.duration}
+                    error={errors.duration?.message}
+                  >
+                    <Select
+                      placeholder="Choose duration"
+                      value={field.value}
+                      menuShouldPortal
+                      options={[
+                        {
+                          value: 3600,
+                          label: '1 hour',
+                        },
+                        {
+                          value: 10800,
+                          label: '3 hours',
+                        },
+                        {
+                          value: 21600,
+                          label: '6 hours',
+                        },
+                        {
+                          value: 43200,
+                          label: '12 hours',
+                        },
+                        {
+                          value: 86400,
+                          label: '24 hours',
+                        },
+                      ]}
```

Comment:
Maybe we could have a const out of component with those static options (and the ones above)

Reply:
yes, we can, but it doesn't make sense a lot because there no others consumers of those consts